src/modules/ekf2/EKF/ekf_helper.cpp

```diff
@@ -972,7 +972,7 @@ void Ekf::resetQuatStateYaw(float yaw, float yaw_variance)
 	}
 
 	// add the reset amount to the output observer buffered data
```

Comment:
The comment is now slightly off and the function name is explicit enough, what about removing it?